deepfence_worker/cronjobs/notification.go

```diff
@@ -103,6 +105,11 @@ func SendNotifications(ctx context.Context, task *asynq.Task) error {
 	wg := sync.WaitGroup{}
 	wg.Add(len(integrations))
 	for _, integrationRow := range integrations {
+		if integrationRow.ErrorMsg.String != "" && time.Since(integrationRow.LastSentTime.Time) < DefaultNotificationErrorBackoff {
```

Comment:
This integration will get blocked permanently I think. It needs to recover if the service is back up again.

Reply:
also this doesn't handle the case when the new integration is added the field `LastSentTime` will be `0`

Reply:
This should be ok... as the `if` check has an "AND" condition.  
* For a new integration, the `ErrorMsg` will be empty.. so the if check will fail and the notifications will be attempted
* For a blocked integration, the `time.Since(integrationRow.LastSentTime.Time)` will be false after the backoff period. So the blocked notifications will be attempted. In this attempt, if there's no error, the previous `ErrorMsg` will be cleared and the integration will become `Active`